homeassistant/components/recorder/statistics.py

```diff
@@ -450,21 +448,26 @@ def no_conversion(val: Any, _: Any) -> float | None:
         else:
             convert = no_conversion
         ent_results = result[meta_id]
-        ent_results.extend(
-            {
-                "statistic_id": statistic_id,
-                "start": _process_timestamp_to_utc_isoformat(db_state.start),
-                "mean": convert(db_state.mean, units),
-                "min": convert(db_state.min, units),
-                "max": convert(db_state.max, units),
-                "last_reset": _process_timestamp_to_utc_isoformat(db_state.last_reset),
-                "state": convert(db_state.state, units),
-                "sum": (_sum := convert(db_state.sum, units)),
-                "sum_increase": (inc := convert(db_state.sum_increase, units)),
-                "sum_decrease": None if _sum is None or inc is None else inc - _sum,
-            }
-            for db_state in group
-        )
+        for db_state in group:
+            start = process_timestamp(db_state.start)
+            end = start + timedelta(hours=1)
```

Comment:
You might want to CONST that one, and maybe in minutes.
It's not unimaginable to think the aggregations period might be configurable in some future ;)

Reply:
Yes, and a start is here: https://github.com/home-assistant/core/pull/56006
Let's worry about improving it in that PR though.